### DISCUSSION TITLE
How to use fixer classes with RectorConfig class?

### DISCUSSION THREAD

**Discussion by lugosium** [rectorphp/rector#7143] [Q&A] [answered]
Hello,

How to use fixer classes with rector 0.12 and the new RectorConfig class please?

Like these one:

```
use PhpCsFixer\Fixer\ClassNotation\OrderedClassElementsFixer;
use PhpCsFixer\Fixer\Import\NoUnusedImportsFixer;
```

Thank you

**Comment by TomasVotruba** [comment, marked as answer]
Hi, these are part of ECS, not the Rector.
You shoud register them in ECS with `ECSConfig` class.

**Comment by lugosium** [comment]
Ah indeed! Thank you very much.
